app/view/button/ShowRedliningToolsPanelController.js

```diff
@@ -0,0 +1,190 @@
+ * class, which was adapted to match the MVVC structure better.
+ *
+ * @class MoMo.client.view.button.ShowRedliningToolsPanelController
+ */
+Ext.define('MoMo.client.view.button.ShowRedliningToolsPanelController', {
+    extend: 'Ext.app.ViewController',
+
+    requires: [
+    ],
+
+    alias: 'controller.button.showredliningtoolspanel',
+
+    /**
+     * Placeholder for the redlining tools panel
+     */
+    btnPanel: null,
+
+    /**
+    *
+    */
+    onToggle: function(btn, pressed){
+        var me = this;
+        if (pressed){
+            me.showRedliningToolsPanel();
+        } else {
+            me.hideRedliningToolsPanel();
+            me.deactivateRedliningTools();
+        }
+    },
+
+    /**
+     * Creates a panel containing buttons for redlining tools.
+     * The position of the panel will be computed dynamically (s. method
+     * #computePosition below).
+     * The `redlinePointStyle`, `redlineLineStringStyle` and
+     * `redlinePolygonStyle` must be declared here by instanciating of the panel
+     * as these will be uses as default values in the styler class (compare
+     * {@link MoMo.client.view.window.RedliningStylerWindow})
+     */
+    createRedliningButtonsPanel: function() {
+
+        var me = this;
+
+        var parentBtn = me.getView().getEl();
+
+        var position = me.computePosition(parentBtn);
+
+        var postitPictureUrl = Ext.Loader.getPath('MoMo.client') +
+            '/../../../resources/img/blue-post-it.png';
+
+        var btnPanel =
+            Ext.create("MoMo.client.view.panel.RedliningToolsPanel", {
+                style: {
+                    'top': position.top,
+                    'right': position.right
+                },
+                bodyStyle: {
+                    background: 'transparent'
+                },
+                postitPictureUrl: postitPictureUrl,
+                redlinePointStyle: new ol.style.Style({
+                    image: new ol.style.Circle({
+                        radius: 7,
+                        fill: new ol.style.Fill({
+                            color: '#5fa2dd'
+                        }),
+                        stroke: new ol.style.Stroke({
+                            color: '#ececec',
+                            width: 2
+                        })
+                    })
+                }),
+                redlineLineStringStyle: new ol.style.Style({
+                    stroke: new ol.style.Stroke({
+                        color: '#ececec',
+                        width: 2
+                    })
+                }),
+                redlinePolygonStyle: new ol.style.Style({
+                    stroke: new ol.style.Stroke({
+                        color: '#ececec',
+                        width: 2
+                    }),
+                    fill: new ol.style.Fill({
+                        color: '#5fa2dd'
+                    })
+                })
+            }
+        );
```

Comment:
Is it possible to move/externalize the config of the `btnPanel` in some way? Maybe as default values in the `RedliningToolsPanel` view? Or is it even possible to make clever use of data binding here, e.g. for the color values!?

If this is too much for know, just ignore this comment (but keep it in mind :wink: )